drivers/timer/hpet.c

```diff
@@ -345,11 +352,16 @@ int sys_clock_driver_init(const struct device *dev)
 	reg = hpet_timer_conf_get();
 	reg &= ~TIMER_CONF_PERIODIC;
 	reg &= ~TIMER_CONF_FSB_EN;
-	reg |= TIMER_CONF_MODE32;
+	reg &= TIMER_CONF_MODE32;
```

Comment:
so you are not setting this config mode then?
Or did you mean to have : reg &= ~TIMER_CONF_MODE32;  ?

Reply:
It's a mistake. fixed.